Example/VimeoUpload-iOS-2Step/VimeoUpload-iOS-2Step/AppDelegate.swift

```diff
@@ -34,7 +34,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool
     {
         AFNetworkReachabilityManager.sharedManager().startMonitoring()
-        
+        NewVimeoUpload.sharedInstance // Ensure init is called on launch
```

Comment:
I don't like the idea of sharedInstance initializers with side effects, perhaps add a method like `setup()`?